.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+jobs:
+  checkCommit:
+    name: Check commit
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    steps:
+    - id: commit
+      run: echo "::set-output name=message::${{ github.event.head_commit.message }}"
+    outputs:
+      commitMsg: ${{ steps.commit.outputs.message }}
+  release:
+    name: Release
+    needs: [checkCommit]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12'
+      - name: Install yarn
+        run: |-
+         NODE_ENV=development yarn
+      - name: Test packages
+        run: yarn test
+      - name: Release packages
+        env:
+          NODE_ENV: production
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: |-
+          echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" >> $HOME/.npmrc 2> /dev/null
```

Comment:
did we do a test release to a personal npm space to ensure this workflow works?

Reply:
The workflow was sourced from `holocron` which has an identical project structure to `one-app-cli` so it should work as expected.